Yoast/wordpress-seo
# Change the height and margin to add space between selection bar and the icon in Gutenberg

## Summary

This PR can be summarized in the following changelog entry:

* Changed the height of the tab links in 'metabox-sidebar' to add space between the selection bar and the actual icon (5px more because of changing the box-sizing to content box in Gutenberg)
* Added margin to the tab links to restore appearance in Gutenberg

## Test instructions

This PR can be tested by following these steps:

* Edit a post in mobile view
* Inspect the tab icons (traffic light, social, and advanced) in Gutenberg and make sure that when selected, the bar and the icon has proper space between them
* The appearance in desktop view should also have enough space between selection bar and icon
* The appearance in the classic editor should not be changed in desktop or mobile view

## Quality assurance


Fixes https://github.com/Yoast/gutenberg/issues/52
